docs/lib/snippets/migrations/runtime_verification.dart

```diff
@@ -3,10 +3,10 @@ import 'package:drift/drift.dart';
 // #docregion
 // import the migrations tooling
 import 'package:drift_dev/api/migrations.dart';
+// import foundation to determine if built with debug mode
+import 'package:flutter/foundation.dart';
```

Comment:
Thanks for the pull request ! But actually, I think this is not going to work because `flutter` is not available in the documentation (and I don't want it to be available, not depending on Flutter makes building it way easier).

As you said, `kDebugMode` is supposed to be the one from Flutter. A consequence of not depending on Flutter is that we can't add the import to the full file. If you want to fix this properly, it would be best to add a fake Flutter package in `docs/assets`. We also do this for [path_provider](https://github.com/simolus3/drift/tree/develop/docs/assets/path_provider) for the exact same reason. Maybe `kDebugMode` is known well enough to not need the explicit import shown in the snippet?